app/scripts/directives/bindService.js

```diff
@@ -139,37 +157,64 @@
       });
     };
 
+    bindFormStep = {
+      id: 'bindForm',
+      label: 'Binding',
+      view: 'views/directives/bind-service/bind-service-form.html',
+      valid: true,
+      onShow: showBind
+    };
+
+    bindParametersStep = {
+      id: 'bindParameters',
+      label: 'Parameters',
+      view: 'views/directives/bind-service/bind-parameters.html',
+      hidden: true,
+      onShow: showParameters
+    };
+
+    resultsStep = {
+      id: 'results',
+      label: 'Results',
+      view: 'views/directives/bind-service/results.html',
+      valid: true,
+      onShow: showResults
+    };
+
+    var updateInstance = function() {
+      if (!ctrl.serviceClasses) {
+        return;
+      }
+
+      var instance = ctrl.target.kind === 'Instance' ? ctrl.target : ctrl.serviceToBind;
+      if (!instance) {
+        return;
+      }
+
+      ctrl.serviceClass = ctrl.serviceClasses[instance.spec.serviceClassName];
+      ctrl.serviceClassName = instance.spec.serviceClassName;
+      ctrl.plan = BindingService.getPlanForInstance(instance, ctrl.serviceClass);
+      ctrl.parameterSchema = _.get(ctrl.plan, 'alphaBindingCreateParameterSchema');
```

Comment:
They don't come up too often, but do you think we ever need to doc these `alpha` kinds of properties with a `TODO` of `FIXME`?  Last I remember was the `osb` prefixes.

Reply:
Definitely reasonable. We've been trying to track the changes in the card because there are so many coming:

https://trello.com/c/y1lbFrbp

Reply:
Nice, that works for me.  Something like `FIXME: API ALPHA` could be a nice way to search, but might not be worth it if already tracked like the Trello.